### PR TITLE
Competition detail page

### PR DESCRIPTION
**Pull request by Tthomas63**
Changes:
 - Initial Style Setup
 - Pass object via context ( For now )

### CODE REVIEW

File: src/apps/competitions/views.py [hunk 1]
```diff
@@ -10,5 +13,16 @@ class CompetitionForm(LoginRequiredMixin, TemplateView):
     template_name = 'competitions/form.html'
 
 
-class CompetitionDetail(LoginRequiredMixin, TemplateView):
+class CompetitionDetail(LoginRequiredMixin, TemplateView, ContextMixin):
     template_name = 'competitions/detail.html'
+
```
Comment (ckcollab) [inline src/apps/competitions/views.py]: You will get competition details via ajax request!

**Comment by Tthomas63** [comment]
Current State:
screen shot 2018-01-28 at 11 03 10 pm
screen shot 2018-01-28 at 11 03 02 pm

**Comment by ckcollab** [comment]
I'm loving it, coming together so nice

**Comment by ckcollab** [comment]
**Current screenshot:** (unless I did something wrong)

image

I would say get rid of the message class everywhere (maybe keep it in a couple spots), let things float out there a bit. Also please make all margins line up (extend to the edge)

-----------

I did a few of these changes and pushed what I did, feel free to undo these changes but i just wanted to push so you could see

**Pull request merged**